### PR TITLE
Refactor device validations imports

### PR DESCRIPTION
**Pull request by fievelk**
Linked to gridsingularity/d3a-interface#230

### CODE REVIEW

**Review by Hassan754 — DISMISSED** [review]
LGTM

File: tests/test_home_meter.py [hunk 1]
```diff
@@ -20,9 +20,9 @@
 from unittest.mock import call
 from unittest.mock import create_autospec
 
-from d3a_interface.device_validator import DeviceValidator
-from d3a_interface.device_validator import HomeMeterValidator
 from d3a_interface.exceptions import D3AException
+# from d3a_interface.validators.base_validator import BaseValidator
```
Comment (hannesdiedrich) [inline tests/test_home_meter.py:24]: You can remove this comment now it think. :-)

**Review by hannesdiedrich — APPROVED** [review]
One minor refactoring, LGTM

**Review by Hassan754 — APPROVED** [review]

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**